src/gc/impl/conservative/gc.d

```diff
@@ -3819,6 +3963,9 @@ struct SmallObjectPool
         // Convert page to free list
         size_t size = binsize[bin];
         void* p = baseAddr + pn * PAGESIZE;
+        size_t biti = pn * (PAGESIZE / 16);
+        base.freebits.set(biti);
```

Comment:
Merge error? BTW: freebits are already maintained correctly during malloc/free (this changed recently). They are supposed to be all set for an unused page.